.pre-commit-config.yaml

```diff
@@ -25,3 +25,7 @@ repos:
         args:
           - --remove-empty-cells
           - --preserve-cell-outputs
+  - repo: https://github.com/pycqa/isort
```

Comment:
I assume this step checks and applies fixes if necessary, right?
Is the order important? meaning if black should run before or after.

Reply:
Hi @mosheraboh,

Thank you for the feedback! 🙌

Yes, the pre-commit step for `isort` both checks and applies fixes as necessary to ensure imports are properly sorted.
As for the order: It's generally recommended that `isort` runs before black, as `isort`organizes imports and `black` formats the entire file. This avoids potential conflicts where `black` might undo some of `isort`'s changes.

I’ve now pushed a commit that updates the order in `.pre-commit-config.yaml` to ensure `isort` runs before `black`.

Reply:
🙏